### PR TITLE
twitterBootstrap3: init at 3.3.7

### PR DESCRIPTION
**Pull request by Profpatsch**
Add version 3 of bootstrap, just the dist (no building from less).

### CODE REVIEW

File: pkgs/development/web/twitter-bootstrap/v3.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "bootstrap-${version}";
+  version = "3.3.7";
+
+  src = fetchurl {
+    url = "https://github.com/twbs/bootstrap/releases/download/v${version}/bootstrap-${version}-dist.zip";
```
Comment (sternenseemann) [inline pkgs/development/web/twitter-bootstrap/v3.nix]: Replace this with `fetchFromGitHub` which doesn't break if the URL changes.

File: pkgs/development/web/twitter-bootstrap/v3.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  buildInputs = [ unzip ];
+
+  dontBuild = true;
+  installPhase = ''
+    mkdir $out
+    cp -r * $out/
+  '';
+
+  meta = {
```
Comment (sternenseemann) [inline pkgs/development/web/twitter-bootstrap/v3.nix]: Add a maintainer?

Comment (Profpatsch) [inline pkgs/development/web/twitter-bootstrap/v3.nix]: I won’t add myself. But help yourself. :)

File: pkgs/development/web/twitter-bootstrap/v3.nix (hunk 1 above)
Comment (Profpatsch) [inline pkgs/development/web/twitter-bootstrap/v3.nix]: Doesn’t work, is no release accessible from there.

**Pull request merged**